api/envoy/api/v2/route/route.proto

```diff
@@ -609,6 +609,10 @@ message RouteAction {
   CorsPolicy cors = 17;
 
   reserved 21;
+
+  // When the request is a gRPC request, use the 'grpc-timeout' header, or it's default value
+  // (infinity) instead of 'timeout'.
```

Comment:
Can we link to RFC here in comment?

Reply:
It is an expired internet draft, but it is archived by IETF, so it should be a stable reference. AFAIK the contents match that in https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md. Not sure which we should refer to, thoughts?